src/components/Article.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
```

Comment:
You can add an article tag instead of an empty tag
```suggestion
  <article>
    <h1>{title}</h1>
    <span>{date}</span>
    <p>{text}</p>
  </article>
```

Reply:
+